### ISSUE TITLE
warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"

### ISSUE THREAD

**Issue by adityaU** [potatosalad/erlang-libdecaf#6]
```
Compiling libdecaf
make: Entering directory '/root/.afterglow/deps/libdecaf/c_src'
make[1]: Entering directory '/root/.afterglow/deps/libdecaf/.ed448goldilocks'
cc -std=c99 -fno-strict-aliasing -pedantic -Wall -Wextra -Werror -Wunreachable-code -Wmissing-declarations -Wunused-function -Wno-overlength-strings  -Isrc/include -Isrc/GENERATED/include -Isrc/GENERATED/c -O2 -maes -mavx2 -mbmi2
-ffunction-sections -fdata-sections -fvisibility=hidden -fomit-frame-pointer -fPIC  -c -o build/obj/shake.o src/shake.c
In file included from /usr/include/assert.h:35:0,
                 from src/shake.c:16:
/usr/include/features.h:148:3: error: #warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE" [-Werror=cpp]
 # warning "_BSD_SOURCE and _SVID_SOURCE are deprecated, use _DEFAULT_SOURCE"
   ^~~~~~~
cc1: all warnings being treated as errors
Makefile:259: recipe for target 'build/obj/shake.o' failed
make[1]: *** [build/obj/shake.o] Error 1
make[1]: Leaving directory '/root/.afterglow/deps/libdecaf/.ed448goldilocks'
Makefile:182: recipe for target '/root/.afterglow/deps/libdecaf/.ed448goldilocks/build/lib/libdecaf.so.1' failed
make: *** [/root/.afterglow/deps/libdecaf/.ed448goldilocks/build/lib/libdecaf.so.1] Error 2
make: Leaving directory '/root/.afterglow/deps/libdecaf/c_src'
===> Hook for compile failed!
```

compiler info: 

```
cc -v
Using built-in specs.
COLLECT_GCC=cc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/6/lto-wrapper
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Debian 6.3.0-18+deb9u1' --with-bugurl=file:///usr/share/doc/gcc-6/README.Bugs --enable-languages=c,ada,c++,java,go,d,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-6 --program-prefix=x86_64-linux-gnu- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-libmpx --enable-plugin --enable-default-pie --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-6-amd64/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-6-amd64 --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-6-amd64 --with-arch-directory=amd64 --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --with-target-system-zlib --enable-objc-gc=auto --enable-multiarch --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 6.3.0 20170516 (Debian 6.3.0-18+deb9u1)
```

**Comment by adityaU** [comment]
fixed by removing -Werror from Makefile. is there any other way to fix this?

**Comment by potatosalad** [comment]
This should be fixed as part of [`libdecaf` version 2.0.0](https://hex.pm/packages/libdecaf/2.0.0).  Please re-open if the problem still persists.
